packages/deprecation-crawler/src/lib/output-formatters/markdown/deprecation-index.md.formatter.ts

```diff
@@ -0,0 +1,61 @@
+import { CrawlConfig, CrawledRelease, Deprecation } from '../../models';
+import { formatCode } from '../../utils';
+import { writeFileSync } from 'fs';
+import * as path from 'path';
+import { EOL } from 'os';
+
+export async function generateDeprecationIndex(
+  config: CrawlConfig,
+  crawledRelease: CrawledRelease
+): Promise<void> {
+  console.log('📝 Update deprecation index markdown format');
+
+  const deprecationsByFileAndTag = crawledRelease.deprecations
+    .map((d) => ({ ...d, version: d.version ? d.version : crawledRelease.tag }))
+    .reduce((tags, deprecation) => {
+      return {
+        ...tags,
+        [deprecation.version]: [
+          ...(tags[deprecation.version] || []),
+          deprecation,
+        ],
+      };
+    }, {});
+
+  const filePath = path.join(config.outputDirectory, 'deprecation-index.md');
+
+  const newlines = '';
+
+  const updatedSections = [
+    '# Deprecations Index',
+    newlines,
+    await getDeprecationList(deprecationsByFileAndTag),
+  ];
+
+  const newMd = updatedSections.join(EOL + EOL) + newlines;
+  writeFileSync(filePath, formatCode(newMd, 'markdown'));
+
+  console.log('Updated group-based markdown format');
+}
+
+async function getDeprecationList(groupedDeprecations: {
+  [version: string]: Deprecation[];
+}): Promise<string> {
+  return Object.entries(groupedDeprecations)
+    .map(([version, deprecations]) => {
+      const treeName = deprecations[0].version;
```

Comment:
Note that `version` can be untagged (empty).
In #84 this was fixed for another formatter where we did `deprecation.version || tag`.

Reply:
https://github.com/timdeschryver/deprecation-manager/pull/85/commits/d92cc2d91f23e4944cf167ffe58691567b745c10